pkgs/games/steam/fhsenv.nix

```diff
@@ -271,10 +271,10 @@ in buildFHSEnv rec {
   '' + extraProfile;
 
   runScript = writeShellScript "steam-wrapper.sh" ''
-    if [ -f /host/etc/NIXOS ]; then   # Check only useful on NixOS
-      ${glxinfo-i686}/bin/glxinfo >/dev/null 2>&1
+    if [ -f /etc/NIXOS ]; then   # Check only useful on NixOS
+      ${glxinfo-i686}/bin/glxinfo 2>&1 | grep -q Error
       # If there was an error running glxinfo, we know something is wrong with the configuration
-      if [ $? -ne 0 ]; then
+      if [ $? -eq 0 ]; then
```

Comment:
> glxinfo does not write a proper error code on errors 🤦

Does this mean that glxinfo's exit code is 0 even when it encounters unrecoverable errors?

Reply:
Exactly 😅